instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/server/ServerSpan.java

```diff
@@ -23,13 +26,7 @@ public final class ServerSpan {
    */
   @Nullable
   public static Span fromContextOrNull(Context context) {
```

Comment:
I believe we have a couple of customers using `ServerSpan.fromContextOrNull` to find the server span. Should we attempt not to break this? Maybe instead of relying on span keys we should instrument this method when it is bundled in user code?

Reply:
I don't think it's a breaking change - we're not using the `SpanKey`s anymore, that's true, but the local root logic should be equivalent to that. We should be fine as long as we don't remove that class too early.

Reply:
I really should have asked this in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/5779 Let me rephrase. If user has an old version of api where
```java
public static Span fromContextOrNull(Context context) {
  return SpanKey.SERVER.fromContextOrNull(context);
}
```
in their application then would it still work after #5779 and this change? If previously span was stored under `SpanKey.KIND_SERVER` is it still stored there besides the new keys?

Reply:
Oh, I see, like using the old instrumentation-api with a new javaagent... no, I suppose not - the `SERVER` span key was removed after all.
That can be fixed though, by adding a new context bridge that'll connect the old `SERVER` key with the local root span.

Reply:
That would we awesome.